mmdet/core/mask/structures.py

```diff
@@ -0,0 +1,423 @@
+import mmcv
+import numpy as np
+import pycocotools.mask as maskUtils
+import torch
+
+
+class Masks(object):
+
+    def rescale(self, scale, interpolation='nearest'):
+        pass
+
+    def resize(self, h, w, interpolation='nearest'):
+        pass
+
+    def flip(self, flip_direction):
+        pass
+
+    def pad(self, h, w, pad_val):
+        pass
+
+    def crop(self, bbox):
+        pass
+
+    def crop_and_resize(self,
+                        bboxes,
+                        mask_h,
+                        mask_w,
+                        assigned_inds,
```

Comment:
Rename it to `inds`. The interface of masks should not be correlated to the assignment.